src/index.html

```diff
@@ -2,16 +2,145 @@
+          Name:
+          <input id="name" type="text">
+        </label>
+        <label class="label"  for="number">
+          How old are You ?
+          <input class="form-number"
+            type="number"
+            id="number"
+            value="12"
+            min="0"
+            max="100"
+          >
+        </label>
+        <label class="label"  for="formdate"> Full date of birth
+          <input
+           id="formdate"
+           type="date"
+           name="formdate"
+          >
+        </label>
+        <label class="label2" for="accept">
+          I accept the term of the agreement
+          <input id="accept" type="checkbox">
+        </label>
+      </fieldset>
+      <fieldset class="form-field">
+        <legend>Registration</legend>
+        <label class="label" for="email">
+           E-mail:
+          <input
+           id="email"
+           type="email"
+           placeholder="email@example.com"
+          >
+        </label>
+        <label class="label" for="password">
+          Password: <input id="password" type="password">
+        </label>
+      </fieldset>
+      <fieldset class="form-field">
+        <legend>An interesting fact about you!</legend>
+        <label class="label">
+          <label
+          for="Cats"
+          class="label"
+          >
+          Do you love cats?
+            <input
+              id="Cats"
+              type="radio"
+              name="Cats"
+            > Yes
+          </label>
+          <label class="label labelcats" for="Cats-2">
+            <input
+            id="Cats-2"
+            type="radio"
+            name="Cats"
+            >
+            No
+          </label>
+        </label>
+        <label class="label" for="color">
+          What is your favorite color?
+          <input
+          id="color"
+          type="color"
+          >
+        </label>
+        <label class="label" for="datelocal">
+          What time do you go to bed?
+          <input
+           id="datelocal"
+           type="datetime-local"
+          >
+        </label>
+        <label class="label" for="Cars" >
+          What are your favorite brand of cars?
+          <select
+           name="Car"
+           required multiple
+           id="Cars">
+            <option value="1">BMW</option>
+            <option value="2">Audi</option>
+            <option value="3">Lada</option>
+          </select>
+        </label>
+        <label class="label" for="form-work">
+          How do you rate our work?
+          <input id="form-work" type="range"
+           min="1"
+           max="100"
+           value="1"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="form-field">
+        <legend>Additional info:</legend>
+        <label class="label"  for="textarea">
+         Comments
+          <textarea
+          id="textarea"
+          name="Comments"
+          cols="20"
+          rows="2"
+          >
+          </textarea>
```

Comment:
```suggestion
          <textarea
          id="textarea"
          name="Comments"
          cols="20"
          rows="2"
          ></textarea>
```
To fix it 
![image](https://user-images.githubusercontent.com/97547824/212394415-193c75f4-7488-418a-9f76-d70b71952a7a.png)

Reply:
Когда я тег textarea делаю не на одной линии, то у меня линт тесты не проходят. По этому я не изменял textarea. Возможно есть другое решение?
Спасибо за ответ Ваш и проверки. С уважением, Виталик Поверенов.